File: Python/Mahasiswa.py
from SivitasAkademik import SivitasAkademik

class Mahasiswa(SivitasAkademik):

    __NIM = ""
    __nama = ""
    __jenis_kelamin = ""
    __fakultas = ""
    __prodi = ""

    def __init__(self, NIM = "", nama = "", jenis_kelamin = "",
                fakultas = "", prodi = ""):
        self.__NIM = NIM
        self.__nama = nama
        self.__jenis_kelamin = jenis_kelamin
        self.__fakultas = fakultas
        self.__prodi = prodi

    def get_NIM(self):
        return self.__NIM

    def set_NIM(self, NIM):
        self.__NIM = NIM

    def get_nama(self):
        return self.__nama

    def set_nama(self, nama):
        self.__nama = nama

    def get_jenis_kelamin(self):
        return self.__jenis_kelamin

    def set_jenis_kelamin(self, jenis_kelamin):
        self.__jenis_kelamin = jenis_kelamin

    def get_fakultas(self):
        return self.__fakultas

    def set_fakultas(self, fakultas):
        self.__fakultas = fakultas

    def get_prodi(self):
        return self.__prodi

    def set_prodi(self, prodi):
        self.__prodi = prodi

File: Java/Mahasiswa.java
class Mahasiswa extends SivitasAkademik
{
    private String NIM;
    private String nama;
    private String jenis_kelamin;
    private String fakultas;
    private String prodi;

    public Mahasiswa()
    {
        this.NIM = "";
        this.nama = "";
        this.jenis_kelamin = "";
        this.fakultas = "";
        this.prodi = "";
    }

    public String getNIM(){
        return this.NIM;
    }
    public void setNIM(String NIM){
        this.NIM = NIM; 
    }

    public String getnama(){
        return this.nama;
    }
    public void setnama(String nama){
        this.nama = nama; 
    }

    public String getfakultas(){
        return this.fakultas;
    }
    public void setfakultas(String fakultas){
        this.fakultas = fakultas; 
    }

     public String getprodi(){
        return this.prodi;
    }
    public void setprodi(String prodi){
        this.prodi = prodi; 
    }

    public String getjenis_kelamin(){
        return this.jenis_kelamin;
    }
    public void setjenis_kelamin(String jenis_kelamin){
        this.jenis_kelamin = jenis_kelamin;
    }

}